SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialPackageMapClient.h

```diff
@@ -56,12 +57,17 @@ class SPATIALGDK_API USpatialPackageMapClient : public UPackageMapClient
 	// Expose FNetGUIDCache::CanClientLoadObject so we can include this info with UnrealObjectRef.
 	bool CanClientLoadObject(UObject* Object);
 
+	bool IsEntityPoolReady() const;
+
 	virtual bool SerializeObject(FArchive& Ar, UClass* InClass, UObject*& Obj, FNetworkGUID *OutNetGUID = NULL) override;
 
 private:
 	UPROPERTY()
 	USpatialClassInfoManager* ClassInfoManager;
 
+	UPROPERTY()
+	UEntityPool* EntityPool;
+
```

Comment:
Does the package map client actually need the net driver? Looks like it could be removed also.

Reply:
Thought it was not possible, but then fixed it with some magic :)